app/src/screens/PersonCredential.tsx

```diff
@@ -104,7 +104,7 @@ const PersonCredentialScreen: React.FC = () => {
     <SafeAreaView style={styles.pageContainer}>
       <ScrollView contentContainerStyle={[styles.pageContent]}>
         <View>
-          <View style={[styles.container]}>
+          {/* <View style={[styles.container]}>
```

Comment:
Need to update OCA bundle resolver to return branding for a credential that can be supplied to the credential card component without a credential exchange record. Otherwise this is hardcoded to look like the the 1.0 branded credential so it has been omitted.